lib/sales_analyst.rb

```diff
@@ -0,0 +1,22 @@
+require_relative 'sales_engine'
+require'pry'
+
+class SalesAnalyst
+  attr_reader :se
+
+  def initialize(se)
+    @se = se
+  end
+
+  def average_items_per_merchant
+    total_merchant_count = se.merchants.merchants.count
+    total_items_count    = items_per_merchant.reduce(:+).to_f
+    average              = total_items_count / total_merchant_count
+    return average.round(2)
```

Comment:
Moving .to_f to total_items_count from average, changed the return value from 2.0 to 2.8778... 

Adding .round(2) rounds to => 2.88 to match the value in the spec.